### ISSUE TITLE
3D Dice not Rolling

### ISSUE THREAD

**Issue by AnlashokNa** [DrOgres/tftloop#54]
In Foundry v12, when rolling a skill from the character sheet, 3D dice from Dice So Nice! do not roll. There is no dice rolling noise; the result is sent straight to chat.

**Comment by bithir** [comment]
Code to fix, the question is if the changes should be made backwards compatible with v11.

Change the data for ChatMessage.create from:
```
    let chatOptions = {
        user: game.user.id,
        speaker: ChatMessage.getSpeaker({ actor: actor, token: actor.img }),
        type: CONST.CHAT_MESSAGE_TYPES.ROLL,
        roll: r,
        content: chatHTML
    }

```
to
```
    let chatOptions = {
        user: game.user.id,
        speaker: ChatMessage.getSpeaker({ actor: actor, token: actor.img }),
        rolls: [r],
        content: chatHTML
    }
```
I think that should solve it. Note: if you want to make the code backwards compatible with v11, you need to keep your elements.

Let me know if you want me to do the changes and create a PR.

**Comment by DrOgres** [comment]
fixed in 3.0.1 thanks @bithir  for the assist!
